Hello.java

```diff
@@ -0,0 +1,5 @@
+public class Hello {
+   public  void main() {
```

Comment:
So, now it will only work in the test and not "in production" by a user since they will be trying to run "java Hello" and having the main print.

FYI, you can directly call the main method just like any other method (it's just that usually we let the JVM call it on our behalf) by doing Hello.main(null). (Note, this assumes you are not using any args hence null is okay.)